scripts/test_path_preservation.ps1

```diff
@@ -0,0 +1,95 @@
+refreshenv
```

Comment:
Maybe add a comment at the top about what this test does?

Reply:
Fixed in https://github.com/emscripten-core/emsdk/pull/642/commits/047b74a3760050c7851ef7da4e05f4b73bd1cf49